cargo-shuttle/src/lib.rs

```diff
@@ -641,6 +641,7 @@ impl Shuttle {
         };
 
         // Child process and gRPC client for sending requests to it
+        // TODO: Is child stdout should be handled?
```

Comment:
This should just print to stdout

Reply:
I'm getting the following error for `cargo shuttle run`:

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: runtime client endpoint did not open in time
```

Still investigating, do you have any ideas?

Reply:
Ah, I think it is because I'm not using `git` dependencies for my test project.

Reply:
da38f862bcce4520f213673e51d2d451c5201f78